examples/non-monorepo/README.md

```diff
@@ -1,42 +1,36 @@
-# Turborepo non-monorepo starter
```

Comment:
We should probably keep the old README?

Reply:
Ah,  yes, good catch. I doctored it up a little bit as well now.